hw/mcu/nordic/nrf5340/syscfg.yml

```diff
@@ -25,13 +25,15 @@ syscfg.defs:
             - $notnull
         choices:
             - nRF5340_APP
+
     MCU_APP_CORE:
         description: >
             Constant value always set to 1.  It allows to have common
             packages for network and application core that do have
             some differences depending on which core they are build for.
         value: 1
         restriction: MCU_NET_CORE==0
```

Comment:
isn't the real problem a typo above?  restriction_S_   ?

Reply:
I guess, I just wanted the build to restart so, I found something that needed fixing :-)

Reply:
done.

Reply:
That restriction does not hold truth anymore since MCUs are different for both cores. I am removing it and updated description above.